### ISSUE TITLE
Using gereral Resnet causes the loss to become ’nan‘

### ISSUE THREAD

**Issue by z7d1** [kaidic/LDAM-DRW#20]
Thank you for your great work!
I find the backone in your code isn't the general Resnet. They are very different from [the general](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py).

And I try to replace resnet32 in the paper mentioned with resnet34 but the loss cannot converge and turn to nan finally.

This is bash command I tried (resnet32 has been changed to resnet34 realized by torchvision)
```bash
python cifar_train.py --arch resnet32 --gpu 0 --imb_type exp --imb_factor 0.01 --loss_type LDAM --train_rule DRW
```
Could you please provide further explanation?

**Comment by z7d1** [comment]
My apologize... the loss to become 'nan' is occured by lacking of  `class NormedLinear(nn.Module)`. And the paper said but I didn't notice: 
![image](https://github.com/kaidic/LDAM-DRW/assets/65475003/1a14e468-2dbe-4c50-b24f-890ccde34823)
